src/Twig/NotificationsFilter.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Twig;
+
+use Doctrine\Common\Collections\Collection;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
+
+class NotificationsFilter extends AbstractExtension
+{
+    public function getFunctions()
+    {
+        return [
+            new TwigFunction('getUnreadNotifications', [$this, 'getUnread']),
```

Comment:
It should have be a filter be a filter, **but** this is a bad idead to do like that.
In 5 years, when we will have 1K notification, we will load 1000 notifications from the DB for nothing.

Instead, you need to use a dedicated controller (with a nice SQL query) and `render(controller(xxx))` where you want to return your content https://symfony.com/doc/current/templates.html#embedding-controllers